hack/otel/compose.yaml

```diff
@@ -0,0 +1,27 @@
+name: moby-otel
+
+services:
+
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - 16686:16686
+
+  aspire-dashboard:
+    image: mcr.microsoft.com/dotnet/nightly/aspire-dashboard
+    ports:
+      - 18888:18888
```

Comment:
wondering if we should use `127.0.0.1` for these; just in case someone runs this without considering that mapping the port will default to the public interface. @akerouanton any thoughts?

Reply:
If we're updating; do you think it's better to be explicit (`0.0.0.0:18888:18888`) for these? Might give more of a clue that it's listening to "any" IP (not sure if we changed semantics on IPv4/IPv6 for that, but I think we still consider `0.0.0.0` to be "either IPv4/6")

Reply:
made this more explicit